conf/ceci_dragon2.config

```diff
@@ -0,0 +1,88 @@
+/*
+ * CÉCI DRAGON2 CLUSTER
+ * ====================
+ * This configuration file provides sensible defaults to run Nextflow pipelines
+ * on the CÉCI Dragon2 cluster.
+ *
+ * For more information on the CÉCI Dragon2 cluster, refer to this page of the
+ * wiki:
+ *     https://www.ceci-hpc.be/clusters.html#dragon2
+ */
+params {
+    config_profile_name        = 'CÉCI'
+    config_profile_description = 'CÉCI Dragon2 cluster profiles provided by the GIGA Bioinformatics Team.'
+    config_profile_contact     = 'Martin Grignard (@MartinGrignard)'
+    config_profile_url         = 'https://www.ceci-hpc.be/clusters.html#dragon2'
+}
+
+/*
+ * Resources limitations
+ * ---------------------
+ * These resources limitations are maximum values across all nodes of all
+ * queues. At least one node matches these maximum resources in all of the
+ * available queues.
+ *
+ * For more information on the available nodes, use the `sinfo` command.
+ */
+params {
+    max_cpus   = 32
+    max_memory = 384.GB
+    max_time   = 21.days
+}
```

Comment:
These have been 'replaced' by resourceLimits
 
You can keep these for backwards compability with older pipelines, but you should copy over the same values to the new 'resourceLimits' direcrtive. Exampel: https://github.com/nf-core/configs/blob/138f5730f207ee85f96c9d0cd0dab790ab293c89/conf/adcra.config#L32-L37

Reply:
I put them in both places (I saw a bunch of other configurations doing that) in all three configuration files (`giga.config`, `ceci_dragon2.config` and `ceci_nic5.config`).
Just out of curiosity, since the `resourceLimits` directive was only recently added (version `24.04.0` of Nextflow), is there a mechanism that ensures compatibility for older Nextflow versions?

Indeed, on some of the clusters I added, the provided versions are quite old (version `21.08.0` on NIC5).

Reply:
Ah my bad! I'm used to them being at the top and must have missed it :sweat_smile: 

Having resourceLimits doesn't make a difference to older pipeline/Nextflow versions, but older pipeline versions will continue to use the `max_*` params (and a custom function in the pipeline code), which is why we've not removed those params. :)